rct229/rules/section5/section5rule31.py

```diff
@@ -0,0 +1,100 @@
+from rct229.rule_engine.rule_base import (
+    RuleDefinitionBase,
+    RuleDefinitionListIndexedBase,
+)
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.utils.assertions import assert_, getattr_
+from rct229.utils.jsonpath_utils import find_all
+from rct229.utils.match_lists import match_lists_by_id
+
+MANUAL_CHECK_MSG = "SURFACE IN P-RMR HAS SUBSURFACES MODELED WITH DIFFERENT MANUAL SHADE STATUS. VERIFY IF SUBSURFACES MANUAL SHADE STATUS IN B-RMR ARE MODELED THE SAME AS IN P-RMR"
```

Comment:
In my opinion, this message should not be in all caps.  It only makes the message harder to read.
I understand that this is how it appears in the RDS.